debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugRunner.kt

```diff
@@ -74,20 +73,22 @@ class RsDebugRunner : AsyncProgramRunner<RunnerSettings>() {
             cmd.copy(command = "build", additionalArguments = commandArguments)
         }
 
-        val runCommand = { executablePath: Path ->
-            GeneralCommandLine(executablePath)
-                .withWorkDirectory(cmd.workingDirectory)
-                .withParameters(executableArguments)
-                .withEnvironment(cmd.environmentVariables.envs)
+        val createGeneralCommandLine = { executablePath: Path ->
```

Comment:
`runCommand` was more meaningful IMO. These was `buildCommand` that builds executable, and `runCommand` that execute produced binary (accepting a path to it). Now `createGeneralCommandLine` tell me only about the type it produces, but not about its meaning